components/button-group/index.jsx

```diff
@@ -102,6 +102,16 @@ const ButtonGroup = (props) => {
 			</fieldset>
 		);
 	}
+	if (props.variant === 'list') {
+		return (
+			<ul
+				className={classNames('slds-button-group-list', props.className)}
+				role="group"
```

Comment:
Group is implicit here and not needed--unlike in base variant

Reply:
Removed.